.gitignore

```diff
@@ -1,7 +1,6 @@
 .*
 
 !.gitignore
-!.scalafmt.conf
```

Comment:
surely removing this didn't allow us to add scalafmt?
this is saying "allow git to see .scalafmt.conf"

Reply:
Okay